cvss/internal/constants.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+const (
```

Comment:
Do you expect these to change often enough to create a whole file which defines them?

Reply:
No, these two just didn't fit into any other file semantically. I'm planning to put common parsing logic here also, so it will be named parse.go along with common parsing stuff